### DISCUSSION TITLE
Welcome to pglet Discussions!

### DISCUSSION THREAD

**Discussion by FeodorFitsner** [pglet/pglet#123] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kaliasgoldmedal** [comment]
Hai

I am Dr.M.Kaliamoorthy Professor in Electrical Engineering from India.

Thanks for providing us a wonderful tool. I have started using it for an application and It works really excellent.

I am using grid tool in my application. I need to change the color of each row based on some condition.

Can you please help me to do this?

Thanks

**Reply by FeodorFitsner** [reply]
Hello,

Thanks for trying Pglet!

At the moment there is no way to alternate coloring of Grid rows. We are currently focusing on Flet (https://flet.dev/) which is a pivot from Pglet and uses Flutter to render app's UI. We are going to add Grid control into Flet soon and then look into rows coloring.

Thank you!

**Reply by ae5au** [reply]
Sorry to hijack this thread, but I'm curious if you plan to build PowerShell support into Flet.  I like Python, but there are a lot of advantages to PowerShell for some internal IT tools when supporting Windows environments.

**Reply by FeodorFitsner** [reply]
Absolutely, we are going to support PowerShell (which is basically made on top of C# library). But this time we decided to more or less stabilize API and only then start adding new languages. So far we have working "cores" for JavaScript/TypeScrypt, C# and PowerShell. Porting those clients from Pglet to Flet is a matter of implementing control classes with their props and events, but in most cases it's copy-paste :)

**Reply by ae5au** [reply]
Awesome!  I'll start playing around with Flet with Python and use pglet for now when I need PowerShell support.

**Comment by kaliasgoldmedal** [comment]
Hi

Thanks for your reply.

Looking forward to work with Flet soon.
*With Regards*
*Dr.M.Kaliamoorthy B.E.,M.Tech.,Ph.D.,MISTE.,MIEEE*
*Associate Professor *
*Department of EEE*
*Dr.Mahalingam College of Engineering and Technology*
*Pollachi, Coimbatore-642003*
*For More informations please visit *
*www.kaliasgoldmedal.yolasite.com <http://www.kaliasgoldmedal.yolasite.com>*
***@***.*** ***@***.***>*

**Comment by kaliasgoldmedal** [comment]
Hi
I would like to know the status of  alternate coloring of Grid rows in
pglet.
*With Regards*
*Dr.M.Kaliamoorthy B.E.,M.Tech.,Ph.D.,MISTE.,MIEEE*
*Associate Professor *
*Department of EEE*
*Dr.Mahalingam College of Engineering and Technology*
*Pollachi, Coimbatore-642003*
*For More informations please visit *
*www.kaliasgoldmedal.yolasite.com <http://www.kaliasgoldmedal.yolasite.com>*
***@***.*** ***@***.***>*
